multicluster/test/e2e/main_test.go

```diff
@@ -66,6 +66,7 @@ func testMain(m *testing.M) int {
 	flag.StringVar(&testOptions.eastClusterKubeConfigPath, "east-cluster-kubeconfig-path", path.Join(homedir, ".kube", "east"), "Kubeconfig Path of the east cluster")
 	flag.StringVar(&testOptions.westClusterKubeConfigPath, "west-cluster-kubeconfig-path", path.Join(homedir, ".kube", "west"), "Kubeconfig Path of the west cluster")
 	flag.BoolVar(&testOptions.enableGateway, "mc-gateway", false, "Run tests with Multicluster Gateway")
+	flag.BoolVar(&testOptions.kindProvider, "kind", false, "Run tests on kind clusters")
```

Comment:
This is less extensible as it would need one flag for every provider while they are exclusive, could it use same flag format as the antrea's main e2e test?
```
flag.StringVar(&testOptions.providerName, "provider", "vagrant", "K8s test cluster provider")
```

Reply:
Done